### ISSUE TITLE
As a User I would like to be rewarded for Providing or Requesting during "off" hours

### ISSUE THREAD

**Issue by Kaliahh** [P7-Team/P7-WebService#15]
**Is your feature request related to a problem? Please describe.**
How do we make sure that there are always Providers active on the network? It could be that there are times when most people are using their PC, or have some other reason for not Providing, and someone Requesting in that timeframe would experience that nothing is happening. 

**Describe the solution you'd like**
The Service should be able to detect these "off" hours. Whenever one of those comes around, Requesters should pay less to have their Batches worked on and Providers should be payed more.

In other words, the ratio between how much a Requester pays each Provider to work on its Tasks, should be adjusted in favor of the Requester. At the same time, each Provider should do less work to earn Contribution Points.

**Describe alternatives you've considered**
None at the moment
